RUREADEME.md

```diff
@@ -0,0 +1,44 @@
+[![build master](https://github.com/Mari023/AE2WirelessTerminalLibrary/actions/workflows/build_master.yml/badge.svg)](https://github.com/Mari023/AE2WirelessTerminalLibrary/actions/workflows/build_master.yml)
```

Comment:
I think README.ru.md would be better, as it will keep all READMEs together regardless of the language